backend/tests/unit/dal/Services/BuildingServiceTest.cs

```diff
@@ -660,6 +660,84 @@ public void Update_Building_LinkedToProject_Allowed()
             service.Update(building);
         }
 
+        [Fact]
```

Comment:
Buildings didn't have as many tests as parcels, so adding some basic error tests unrelated to my changes.